spec/lib/statements_statistics_spec.rb

```diff
@@ -24,7 +24,7 @@
       ]
       stub_request(:get, 'https://suggestions-store.mysociety.org/export/ca.json')
         .to_return(body: JSON.generate(body))
-      create(:page, position_held_item: 'Q15964890')
+      @page = create(:page, position_held_item: 'Q15964890')
```

Comment:
I think it would be more consistent with the other specs to use a `let(page) { … }` rather than an instance variable here.

Reply:
@mhl Ah sorry, that's my fault, I think it needs to be `let!` rather than `let`, as the page needs to exist before `position_stats.pages` is called in the spec.